### ISSUE TITLE
Question about hyper-parameter settings

### ISSUE THREAD

**Issue by zhourongleiden** [wengwenchao123/DDGCRN#1]
Thanks for your great work and the sharing of the code. I have been recently running the script but I found that some hyper-parameter settings are inconsistent with those specified in the paper. For instance, the epoch and learning rate for PEMSD8 is 300 and 0.003 respectively given in the configuration file, while they are specified as 100 and 0.03 in the paper. May I ask which one should we follow? Thank you.

**Comment by wengwenchao123** [comment]
Hello, thank you for your interest in this work. I suggest following the settings in the configuration file, as it would be best to replicate the effects mentioned in the paper. Although using the settings from the paper can achieve similar results, training for a longer duration may lead to better performance. Therefore, I have set a higher value for the number of epochs. Additionally, to avoid unnecessary training in the later stages, I have implemented an early-stop mechanism in the code. Training will be stopped if there is no improvement after a certain number of epochs. I hope this helps you.
